src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AdjustingInterestRateCurveForWeETHOnArbitrum.md

```diff
@@ -0,0 +1,62 @@
+---
+title: " Adjusting Interest Rate Curve for weETH on Arbitrum"
+author: "ACI"
+discussions: "https://governance.aave.com/t/arfc-adjusting-interest-rate-curve-for-weeth-on-arbitrum/17804"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0xed2fd3dfee1f29f04b6cda4a5c4629fcca32a5c961b1b3e2a49ba6842367ce31"
+---
+
+## Simple Summary
+
+The current proposal suggests adjusting the Interest Rate Curve for weETH on Arbitrum network to align with the mainnet, so Risk Parameters will be the same for weETH on Mainnet, Arbitrum, and Base networks.
+
+The aim is to optimize the utilization rates and improve both lending and borrowing for users.
+
+## Motivation
+
+By adjusting the Interest Rate Curve, we aim to encourage more borrowing and lending activity, optimize the utilization rates, and enhance liquidity on Arbitrum and Base networks.
+
+1. **Base Rate Increase:** Increasing the base rate provides a more attractive return for lenders, encouraging more weETH deposits.
+2. **Slope 1 Adjustment:** Adjusting the Stable Rate Slope offers slightly higher interest for moderate utilization, making borrowing more attractive.
+3. **Slope 2 Adjustment:** Reducing Stable Rate Slope 2 prevents excessively high borrowing costs at higher utilization rates, thus avoiding sharp declines in borrowing activities.
```

Comment:
@marczeller Why are we changing the stable rate slope 1 and 2? It should not matter no as stable rates are disabled or am I missing something?

Reply:
fixed